docs/coordinates/representations.rst

```diff
@@ -48,6 +50,13 @@ between the vertical to the surface at a specific point of the spheroid and its
 projection onto the equatorial plane.
 The latitude is a value ranging from -90 to 90 degrees, the longitude from 0 to 360
 degrees.
+`~astropy.coordinates.BaseBodycentricRepresentation` is the coordinate representation
+recommended by the Cartographic Coordinates & Rotational Elements Working Group
+(see for example its `2019 report <https://rdcu.be/b32WL>`_): the bodicentric latitude
+and longitude are spherical latitude and longitude originated at the barycenter of the
+body, the height is the distance from the spheroid surface on the line of sight.
```

Comment:
Would it be worth adding at the end something like "(i.e., the longitude and height are the same as for the geodetic representations, but the latitude is different)."

Reply:
The height is not really the same: in geodetic coordinates it is measured on a perpendicular to the surface and may have a physical meaning of elevation above the surface, while in bodycentric is more a geometrical quantity (as it is not perpendicular to the surface).

Reply:
I have clarified in the description of the geodetic coordinates.